src/scenes/gameuiscreen.ts

```diff
@@ -68,9 +71,23 @@ export class GameUiScreen extends Phaser.Scene {
             this.load.image(`tile${i}`, `assets/tiles/tile${i}.png`);
         }
         this.load.image("trashCan", "assets/buttons/trashcan.png");
+        // Can't get rid of error message "Object is possibly null"
```

Comment:
Why not? getAll() returns a nullable... same as always. Just make a const, assert and then use. Look in the code for examples.

Reply:
Good then i understood you correctly